src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesController.java

```diff
@@ -53,4 +56,13 @@ public String getMonitoringLocation(
 		}
 		return rtn;
 	}
+
+	@Override
+	public void addCorsMappings(CorsRegistry registry) {
+		registry
+				.addMapping("/**")
+				.allowedOrigins("*")
+				.allowedMethods("GET", "OPTIONS")
+				.allowedHeaders("Origin", "Accept", "X-Requested-With", "Content-Type", "Access-Control-Request-Method", "Access-Control-Request-Headers");
+	}
```

Comment:
Looks good.
 question for my own benefit. Do we need to do this on the other controllers?

Reply:
My question was answered. :)